src/frontend/src/icp/services/ic-transactions.services.ts

```diff
@@ -49,7 +49,7 @@ export const loadNextTransactions = ({
 	identity: OptionIdentity;
 	start?: bigint;
 	maxResults?: bigint;
-	token: IcToken;
+	token: IcToken & IcCanistersStrict;
```

Comment:
isn't it better to create a type somewhere for this?

Reply:
That was actually my first idea, but since the union ended up being needed only twice, I thought it might be too early to introduce a new global type, which might also require some additional Zod logic. Just felt like it was the lighter approach.

Reply:
But we can create one if you wish?